ament_copyright/ament_copyright/parser.py

```diff
@@ -104,7 +99,7 @@ def identify_copyright(self):
             else:
                 self.copyright_identifiers.append(UNKNOWN_IDENTIFIER)
 
-    def parse(self):
+    def parse(self, licenses, known_copyrights):
```

Comment:
My only concern here is that maybe, possibly, somewhere, someone is calling the `parse` function with no arguments. I doubt that is the case, but maybe we want to throw in a deprecation warning / NotImplemented error?

Reply:
That's a fair concern.  What I can do there is to make `licenses` and `known_copyright` set to `None` by default, and if they are unset, get them.  That way any "old" callers will still get the functionality (though it will be slower), and new callers will get the maximum performance.

Reply:
And now fixed in d4ed942